test/distributed/test_store.py

```diff
@@ -0,0 +1,425 @@
+import os
```

Comment:
Assuming this no major change in this file and it's mainly moving test contents around.